.github/workflows/node.js.yml

```diff
@@ -25,6 +25,6 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - run: npm ci
-    - run: npm run start:client
+        cache: 'yarn'
+    - run: yarn
+    - run: yarn start:client
```

Comment:
Doubt [actions/setup-node@v3](https://github.com/actions/setup-node) has node installed

You'll probably have to find another step, but let's try.

I'd also change this step's name to `Use Yarn`